tests/unit/store/traction/ont/getters.spec.js

```diff
@@ -1,15 +1,18 @@
 import { expect, it } from 'vitest'
-import Response from '@/api/v1/Response'
-import { Data } from '@support/testHelper'
 import getters from '@/store/traction/ont/getters'
 import InstrumentFlowcellLayout from '@/config/InstrumentFlowcellLayout'
+import OntInstrumentsFactory from '@tests/factories/OntInstrumentsFactory.js'
+import OntRunsFactory from '@tests/factories/OntRunsFactory.js'
+
+const ontInstrumentsFactory = OntInstrumentsFactory()
+const ontRunsFactory = OntRunsFactory()
 
 describe('getters.js', () => {
   let state, instruments, runs
 
   beforeEach(() => {
-    instruments = new Response(Data.OntInstruments).deserialize.instruments
-    runs = new Response(Data.OntRuns).deserialize.runs
+    instruments = ontInstrumentsFactory.storeData.instruments
```

Comment:
Can you do `{ instruments, runs } = ontInstrumentsFactory.storeData`

Reply:
We may not be able to destructure it this way it as they are from different factories, instruments from ontInstrumentsFactory.storeData and runs from ontRunsFactory.storeData

Reply:
Ah. Didn't notice that.